MacroMate/Conditions/ConditionManager.cs

```
@@ -28,17 +28,22 @@ public void Dispose() {
 
     public CurrentConditions Conditions { get => currentConditions ?? CurrentConditions.Query(); }
 
-    private void OnFrameworkUpdate(IFramework framework) {
+    private void OnFrameworkUpdate(IFramework framework)
+    {
         if (!loggedIn) { return; }
 
-        var newConditions = CurrentConditions.Query();
-        if (currentConditions != newConditions) {
-            currentConditions = newConditions;
-
-            if (ConditionChange != null) {
-                ConditionChange(currentConditions);
+        try
+        {
+            var newConditions = CurrentConditions.Query();
+            if (currentConditions != newConditions)
+            {
+                currentConditions = newConditions;
+                OnConditionChange?.Invoke(currentConditions);
             }
         }
+        catch (Exception)
+        {
+        }
```

This `catch (Exception)` block silently hides any errors. This was erroring because some conditions were broken, by doing this MacroMate will compile and run but many conditions will not actually work.

---

Sorry, I won't do that again.
/em looks away guiltily.
/wait 2
/grovel